src/Form.js

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import FormSection from "./FormSection";
+
+class Form extends React.Component {
+	constructor() {
+		super();
+		this.state = {
+			title: "",
+			userFirstName: "",
+			userLastName: "",
+			callerNumber: "",
+		};
+		this.handleChange = this.handleChange.bind(this);
+	}
+	handleChange(event) {
+		const { name, value } = event.target;
+		this.setState({ [name]: value });
+	}
+	render() {
+		return (
+			<form>
+				<h3>User Information</h3>
+				<label>
+					Title
+					<select value={this.state.title} name="title" onChange={this.handleChange}>
```

Comment:
I like your config idea, I don't think you need it yet, but I would add it to the todo list for good practice